cinema/views.py

```diff
@@ -56,3 +76,58 @@ def get_serializer_class(self):
             return MovieSessionDetailSerializer
 
         return MovieSessionSerializer
+
+    def get_queryset(self):
+        queryset = self.queryset
+        movie_id = self.request.query_params.get("movie")
+        date = self.request.query_params.get("date")
+        if movie_id:
+            movie_id = int(movie_id)
```

Comment:
Consider adding error handling for the conversion of 'movie_id' to an integer. If the query parameter is not a valid integer, it could raise a ValueError.